src/snowflake/snowpark/modin/plugin/extensions/index.py

```diff
@@ -726,10 +760,10 @@ def name(self, value: Hashable) -> None:
         if not is_hashable(value):
             raise TypeError(f"{type(self).__name__}.name must be a hashable type")
         self._query_compiler = self._query_compiler.set_index_names([value])
+        # Update the name of the parent's index only if the parent's current query compiler
```

Comment:
can you update the doc for the name function to make it clear that the inplace replacement only works when no inplace update has been applied to parent

Reply:
done!